### PR TITLE
[DOC] Added Changelog 2.3.0

### PR DESCRIPTION
**Pull request by cpockrandt**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -11,19 +11,74 @@ Library Features
...
+  - Gustaf:
+      - Fixed name conflict (TANDEM)
+
+Infrastructure Updates
+^^^^^^^^^^^^^^^^^^^^^^
+
+  - Added feature to surpress builds of specified apps
+  - Enforce using Python 2.x for documentation
+  - Renaming FindSeqAn to SeqAnConfig in CMake
+  - TODO (h2/rrahn): pkg-config: Define SEQAN_HAS_* flags in pkg-config file (#1938)
```
Comment (cpockrandt) [inline CHANGELOG.rst]: @rrahn ping pong

Comment (h-2) [inline CHANGELOG.rst]: I am currently working on it :)

Comment (rrahn) [inline CHANGELOG.rst]: @h2 how much time do you need? We want to start packaging. 😏

Comment (h-2) [inline CHANGELOG.rst]: please do not do history rewriting while I am working on it :pray:

File: CHANGELOG.rst [hunk 2]
```diff
@@ -11,19 +11,74 @@ Library Features
 
 - Argument Parser:
     - Adds version check support to the argument parser.
-        - Check for new updates of a specific application.
+        - Codeheck for new updates of a specific application.
```
Comment (joergi-w) [inline CHANGELOG.rst]: Codecheck

**Comment by rrahn** [comment]
@cpockrandt can you please update your branch with the changes from hannes and then fix the typos.
Once this is done we can merge it and then start the release

**Comment by h-2** [comment]
@rrahn I pushed directly to @cpockrandt 's branch and I also fixed the typos

**Pull request merged**